exercises/tournament/src/test/java/TournamentTest.java

```diff
@@ -0,0 +1,158 @@
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class TournamentTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void justTheHeaderIfNoInput() {
+        Tournament tournament = new Tournament();
+        assertEquals("Team                           | MP |  W |  D |  L |  P", tournament.printTable());
+    }
+
+    @Test
+    public void aWinIsThreePointsALossIsZeroPoints() throws Exception {
```

Comment:
Why do you have `throws Exception` here? If you're expecting it to throw an exception you should specify exactly which exception you're expecting. But from what I can tell your methods don't throw any exceptions?